tasks/terraform-cli/src/context/azdo-task-context.ts

```diff
@@ -167,6 +170,28 @@ export default class AzdoTaskContext implements ITaskContext {
     get skipExistingWorkspace() {
         return this.getBoolInput("skipExistingWorkspace", false )
     }
+
+    @trackValue("inputs.commands.state.subCommand", isCommand("state"))
+    get stateSubCommand() {
+        return this.getInput("stateSubCommand", true);
+    }
+
+    @trackValue("inputs.commands.state.addresses", isCommandWithSubCommand(["state"], ["list", "rm"], (c) => c.stateSubCommand))

```

Comment:
Indicate is secret to make sure the actual values are not tracked in telemetry.
```suggestion
    @trackValue("inputs.commands.state.addresses", isCommandWithSubCommand(["state"], ["list", "rm"], (c) => c.stateSubCommand), true)

```